docs/action-usage.md

```diff
@@ -18,7 +18,23 @@
 
 ### Copy Standard YAML
 
-If you are after the other job status check provided by Merge Gatekeeper, you will simply need to copy the following YAML.
+<!-- == export: simple-usage / begin == -->
+
+The easiest approach is to copy the standard definition, and save it under `.github/workspaces` directory. There is no further modification required unless you have some specific requirements.
+
+#### With `curl`
+
+```bash
+curl -sSL https://raw.githubusercontent.com/upsidr/merge-gatekeeper/main/example/merge-gatekeeper.yml \
+  > .github/workflows/merge-gatekeeper.yml
+```
+
+#### Directly copy YAML
+
+The below is the copy of [`/example/merge-gatekeeper.yml`](/example/merge-gatekeeper.yml).
+
+<!-- TODO: replace below using Importer once Importer supports code block wrapping
+     == imptr: basic-yaml / begin from: ./example/definitions.yaml#[standard-setup] == -->
 
 ```yaml
 ---
```

Comment:
why don't you change L41-L48 to below?
```
on:
  pull_request:
    branches:
      - main # specify branch(es) where PRs will target
```
listing main and master will be actually no harm. However, repos not recently created don't have main and ppl like me wanted to get rid of in order to avoid other ppl to get confused.

Reply:
I'm not sure what you mean by this, you mean adding a comment to clarify you can specify the target branch?

Reply:
yes, adding a comment to have ppl update the target branch(es).

I think we should make clear where to update and where NOT.
target branch is what we want ppl to confirm and update before installing merge-gatekeeper.

Reply:
Hmm, TBH, I'm not 100% convinced. The clarity with the comment is useful, but it gives the impression that you should be updating the spec. Ideally this should be a "drag-and-drop" kind of solution, where no modification is required at all, you can use it as is. In most cases, it already is, and the only exception is when you have other branching logic such as GitFlow.

It may be good to have a separate doc mentioning that, but not sure to have it as a part of spec that would be copied by users.
We could provide extra comments to explain what each step does, which may be a good middle ground so that it gives clarity, but does not give you the impression that you need to edit unless you have specific use cases for?

Reply:
@teruhiky Ping

Reply:
hmm, i think the template yaml is not complex and we don't need to explain what each step does.
in that case, just keeping it as is would be good enough.

I initially thought ppl might delete "main" from the template yaml if their repo doesn't have main branch.
however, i think it's fine to keep it as is if the comment unintentionally gives an impression ppl need to update the spec.

Reply:
The template YAML is "encouraged" to be used as is, though you can certainly edit as you like. This is because merge-gatekeeper may need to change its spec in the future for better handling for comments, etc.

I'll go ahead with what we have for now, and we can work on better documentation as a separate item.